### DISCUSSION TITLE
While doing model hyperparameter optimization, why seed is being tunned ?

### DISCUSSION THREAD

**Discussion by cahuja1992** [Nixtla/neuralforecast#500] [General]
I can see that for most of the models, the seed is being tunned as well, can anyone help understand that why is that the case ?

**Comment by kdgutier** [comment]
Hey @cahuja1992,

If you want you can tune the random seed to scape bad local optima, by changing the parameters initialization.
The random seed initialization technique has a bigger impact for smaller datasets, for larger datasets the impact is not very big.

It also gives you the possibility to bootstrap the model's initialization by fixing different random seeds.
